### PR TITLE
Fix error on test

### PR DESCRIPTION
**Pull request by elsaalarcon**

### CODE REVIEW

File: one/the_number_is_five.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
-number = 5
-
-if(number == 5)
-  puts "the number is five"
-else
-  puts "the number is NOT five"
+def number_is_five(number)
+  if(number == 5)
```
Comment (geocodinglife) [inline one/the_number_is_five.rb]: Please make this change.
number.eql?(5) ? 'the number is five' : the number is NOT five'

File: one/the_number_is_five.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+    'the number is five'
+  else
+    'the number is NOT five'
+  end
 end
```
Comment (geocodinglife) [inline one/the_number_is_five.rb]: Please remove this line.

File: one/the_number_is_five.rb (hunk 1 above)
Comment (elsaalarcon) [reply one/the_number_is_five.rb]: Done.

File: one/the_number_is_five.rb (hunk 2 above)
Comment (elsaalarcon) [reply one/the_number_is_five.rb]: Done.

**Review by geocodinglife — APPROVED** [review]
Looks good!

**Pull request merged**